### PR TITLE
Create draft-ya-alim-as-sir-minna

### PR DESCRIPTION
**Pull request by thenotepadwriter**
Salaams Adel,

I've created a draft of ya alim as-sir minna.
Source: Qasida Collection (with very minor editing to correct typos)

Can you let me know how to edit the bottom section on the 'order'? Current version is just copied from another Qasida template.

Thanks!
Yomna

### CODE REVIEW

File: site/data/qasida/draft-ya-alim-as-sir-minna
Comment (adehad) [inline site/data/qasida/draft-ya-alim-as-sir-minna:1]: (no need to include the 'draft' in the name, just keep it the intended name: `ya-alim-as-sir-minna.json` - not the `.json` extension, until it gets merged in it is basically a draft anyway!)

File: site/data/qasida/draft-ya-alim-as-sir-minna [hunk 1]
```diff
@@ -0,0 +1,412 @@
...
+
+            ["أَشْكُو إِلَيْكَ وَأَبْكِي"],
+            ["مِنْ شُؤْمِ ظُلْمِي وَإِفْكِي"],
+            ["وَسُوءِ فِعْلِي وَتَرْكِي"],
+            ["وَشَهْوَةِ القِيـلِ وَالقَـالْ"],
+
+            ["وَحُبِّ دُنْيَا ذَمِيمَةْ"],
+            ["مِنْ كُلِّ خَيْرٍ عَقِيمَةْ"],
+            ["فِيهَا البَلَايَا مُقِيمَةْ"],
+            ["وَحَشْوُهَا آفَاتْ وَاشْغَالْ"]
```
Comment (adehad) [inline site/data/qasida/draft-ya-alim-as-sir-minna:56]:
```suggestion
            ["وَحَشْوُهَا آفَاتْ وَاشْغَالْ"],
```

All these verses should end with a comma except the very last one in the list

File: site/data/qasida/draft-ya-alim-as-sir-minna [hunk 2]
```diff
@@ -0,0 +1,412 @@
...
+
+            ["يَا وَيْحَ نَفْسِي الغَوِيَّةْ"],
+            ["عَنِ السَّبِيلِ السَّوِيَّةْ"],
+            ["أَضْحَتْ تُرَوِّجْ عَلَيَّهْ"],
+            ["وَقَصْدُهَا الجَاهُ وَالـمَالْ"]
```
Comment (adehad) [inline site/data/qasida/draft-ya-alim-as-sir-minna:61]: so this one too (and the rest)

File: site/data/qasida/draft-ya-alim-as-sir-minna [hunk 3]
```diff
@@ -0,0 +1,412 @@
...
+
+            ["wa ṣalli fī kulli ḥālah"],
+            ["ʿalā muzīliḍ ḍalālah"],
+            ["man kallamathul ghazālah"],
+            ["muḥammadil haādiyil ddāl"],
+
+            ["wal ḥamdu ililh shukrā"],
+            ["ʿalā niʿam minhu tatrā"],
+            ["naḥmadhu sirran wa jahrā"],
+            ["wa bil ghadāyā wal 'āṣāl"],
```
Comment (adehad) [inline site/data/qasida/draft-ya-alim-as-sir-minna:228]: as this is the last on in the list it should not have a comma

```suggestion
            ["wa bil ghadāyā wal 'āṣāl"]
```

File: site/data/qasida/draft-ya-alim-as-sir-minna [hunk 4]
```diff
@@ -0,0 +1,412 @@
...
+            ["upon he who effaced misguidance,"],
+            ["he to whom the gazelle spoke,"],
+            ["Muhammad, the guide, the leader"],
+
+            ["And all praise is due to Allah along with gratitude"],
+            ["for all His blessings that never cease."],
+            ["We praise Him privately and publicly"],
+            ["by morning and night."],
+
+
```
Comment (adehad) [inline site/data/qasida/draft-ya-alim-as-sir-minna:343]:
```suggestion
            ["by morning and night."]
```

Similarly here as it is the last one should not have a comma

File: site/data/qasida/draft-ya-alim-as-sir-minna [hunk 5]
```diff
@@ -0,0 +1,412 @@
...
+                {"repeat": 1, "index": 33 },
+                {"repeat": 1, "index": 34 },
+                {"repeat": 1, "index": 35 }
+            ],
+            [
+                {"repeat": 1, "index": 36 },
+                {"repeat": 1, "index": 37 },
+                {"repeat": 1, "index": 38 },
+                {"repeat": 1, "index": 39 }
+            ]
```
Comment (adehad) [inline site/data/qasida/draft-ya-alim-as-sir-minna:409]: This is perfectly fine don't worry, I should probably add some instructions, but basically each line in the lists above gets a number from 0
image

So the only change we would need to add here is to add extra entries for the remaining line, and they are grouped by verse sung
```suggestion
            ],
            [
                {"repeat": 1, "index": 40 },
                {"repeat": 1, "index": 41 },
                {"repeat": 1, "index": 42 },
                {"repeat": 1, "index": 43 }
            ],
            [
                {"repeat": 1, "index": 44 },
                {"repeat": 1, "index": 45 },
                {"repeat": 1, "index": 46 },
                {"repeat": 1, "index": 47 }
            ],
            [
                {"repeat": 1, "index": 48 },
                {"repeat": 1, "index": 49 },
                {"repeat": 1, "index": 50 },
                {"repeat": 1, "index": 51 }
            ],
            [
                {"repeat": 1, "index": 52 },
                {"repeat": 1, "index": 53 },
                {"repeat": 1, "index": 54 },
                {"repeat": 1, "index": 55 }
            ],
            [
                {"repeat": 1, "index": 56 },
                {"repeat": 1, "index": 57 },
                {"repeat": 1, "index": 58 },
                {"repeat": 1, "index": 59 }
            ],
            [
                {"repeat": 1, "index": 60 },
                {"repeat": 1, "index": 61 },
                {"repeat": 1, "index": 62 },
                {"repeat": 1, "index": 63 }
            ],
            [
                {"repeat": 1, "index": 64 },
                {"repeat": 1, "index": 65 },
                {"repeat": 1, "index": 66 },
                {"repeat": 1, "index": 67 }
            ],
            [
                {"repeat": 1, "index": 68 },
                {"repeat": 1, "index": 69 },
                {"repeat": 1, "index": 70 },
                {"repeat": 1, "index": 71 }
            ],
            [
                {"repeat": 1, "index": 72 },
                {"repeat": 1, "index": 73 },
                {"repeat": 1, "index": 74 },
                {"repeat": 1, "index": 75 }
            ],
            [
                {"repeat": 1, "index": 76 },
                {"repeat": 1, "index": 77 },
                {"repeat": 1, "index": 78 },
                {"repeat": 1, "index": 79 }
            ],
            [
                {"repeat": 1, "index": 80 },
                {"repeat": 1, "index": 81 },
                {"repeat": 1, "index": 82 },
                {"repeat": 1, "index": 83 }
            ],
            [
                {"repeat": 1, "index": 84 },
                {"repeat": 1, "index": 85 },
                {"repeat": 1, "index": 86 },
                {"repeat": 1, "index": 87 }
            ]
```

**Review by adehad — CHANGES_REQUESTED** [review]
Wa Salaams, MashAllah, and Jummah Mubarak.

A few changes to the file I've added comments in to help, and you will also need to add a page that displays the Qasida, here is an example:

In a new file with location: `site\content\Qasidas\Ya-Alim-As-Sir-Minna.md`
```
# Ya Alim as-Sir Minna - O Knower of our secrets

## Verses

{{% get_title file="qasida/ya-alim-as-sir-minna" lang="en" %}}

{{< verse file="qasida/ya-alim-as-sir-minna" lang="en" >}}

## About

### Author

### Sources

- Arabic, English Translation and Transliteration: [Qasida Collection](https://www.qasidacollection.com/qasida/ilZqHsA8GKrqKDmzoyWS)

```

**Comment by thenotepadwriter** [comment]
Jumma Mubaraka to you too!

Thanks for the really helpful comments, I think they're all integrated now, and I created a pull request for the display page too. I'm not sure if it's quite right now, though?

**Comment by adehad** [comment]
> Jumma Mubaraka to you too!
> 
> Thanks for the really helpful comments, I think they're all integrated now, and I created a pull request for the display page too. I'm not sure if it's quite right now, though?

If we can include that in this pull request that would be ideal!

File: site/data/qasida/ya-alim-as-sir-minna.json [hunk 6]
```diff
@@ -0,0 +1,484 @@
...
+            ["of the ills of my injustice and falsehood"],
+            ["and the evil of my deeds and my abandonment"],
+            ["and of my whimsical speech"],
+
+            ["and (I complain) of love of this blameworthy world"],
+            ["that is barren of any goodness"],
+            ["In it all tribulations reside,"],
+            ["and it is filled with tribulations and preoccupation"],
+
+            ["O Woe to my soul that decieves"],
```
Comment (adehad) [inline site/data/qasida/ya-alim-as-sir-minna.json:283]:
```suggestion
            ["O Woe to my soul that deceives"],
```

(the error on the build is due to spelling of this)

**Comment by thenotepadwriter** [comment]
> > Jumma Mubaraka to you too!
> > Thanks for the really helpful comments, I think they're all integrated now, and I created a pull request for the display page too. I'm not sure if it's quite right now, though?
> 
> If we can include that in this pull request that would be ideal!

Ok, yes - I think it's included in this PR now?

**Comment by adehad** [comment]
Yep perfect, JazakumAllah!

**Comment by thenotepadwriter** [comment]
Awesome, thanks for helping me through it! I get the process now for next time too isA

**Pull request merged**